OKTA-620014 Updates to Web App with Redirect Quickstart (#4295)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/sign-into-web-app-redirect/main/python/defineroute.md

> When you [created an app integration in the admin console](#create-an-app-integration-in-the-admin-console), you set the sign-in redirect URL to <StackSnippet snippet="signinredirecturi" inline /> and the sign-out redirect URL to <StackSnippet snippet="signoutredirecturi" inline />.

Reviewer: ```suggestion
When you [created an app integration in the Admin Console](#create-an-app-integration-in-the-admin-console), you set the sign-in redirect URL to <StackSnippet snippet="signinredirecturi" inline /> and the sign-out redirect URL to <StackSnippet snippet="signoutredirecturi" inline />.
```

1. Open `app.py`.
1. Add a route handler for the sign-in callback handler URI to the end of the file:

   ```py
   @app.route("/authorization-code/callback")
   def callback():
       headers = {'Content-Type': 'application/x-www-form-urlencoded'}
       code = request.args.get("code")
       app_state = request.args.get("state")
       if app_state != session['app_state']:
           return "The app state does not match"
       if not code:

>                return "The code was not returned or is not accessible", 403

Reviewer: ```suggestion
               return "The code wasn't returned or isn't accessible", 403
```
